### DISCUSSION TITLE
how to load OSD belonging to different Ceph cluster

### DISCUSSION THREAD

**Discussion by manju-rn** [rook/rook#13946] [Q&A]
My VM containing the primary ceph cluster crashed completely. So the VM was wiped off and re-installed with microk8s with rook-ceph.
The original disk that was attached (sdb) to earlier ceph cluster still has the data.  However, the new Ceph cluster is unable to load up the OSD - how can I load up the OSD from earlier cluster (which is not accessible) to the newly setup rook-ceph cluster?

```
2024-03-17 04:19:40.051202 D | cephosd: {
    "0d5ffa8e-82cd-40bb-9162-fb013a0a1fd6": {
        "ceph_fsid": "7ac87e1f-3149-4e72-b082-4e0e21c6f212",
        "device": "/dev/sdb",
        "osd_id": 0,
        "osd_uuid": "0d5ffa8e-82cd-40cc-9082-fb013a0a1fd6",
        "type": "bluestore"
    }
}
2024-03-17 04:19:40.051321 I | cephosd: skipping osd.0: "0d5ffa8e-82cd-40cc-9082-fb013a0a1fd6" belonging to a different ceph cluster "7ac87e1f-3149-4e72-b082-4e0e21c6f212"
```

**Comment by microyahoo** [comment]
Maybe you can reference this document: https://rook.io/docs/rook/latest/Troubleshooting/disaster-recovery/#adopt-an-existing-rook-ceph-cluster-into-a-new-kubernetes-cluster
